README.md

```diff
@@ -64,6 +64,7 @@ invoked periodically via [Cloud Scheduler][cloud-scheduler].
       --image "us-docker.pkg.dev/gcr-cleaner/gcr-cleaner/gcr-cleaner" \
       --region "us-central1" \
       --timeout "60s"
+      [--set-env-vars="CONCURRENCY=N"]
```

Comment:
i wouldn't even include it here because its optional

Reply:
Cool, yeah, OK